Fixed spelling mistakes (#197)
codatio/codat-docs: docs/underwriting/uw-decision.md

The [LoanUnderwriter](https://dev.azure.com/codat/Codat%20Spikes/_git/DemosUnderwriting?path=/Codat.Demos.Underwriting.Api/Services/LoanUnderwriter.cs&version=GBmain) service then checks how these values compare to the thresholds set in the app: 

1. Gross profit margin must be more than `MinGrossProfitMargin` threshold set to 0.4,
2. Revenue must exceed the `RevenueThreshold` set to 0.3, and
3. Gearing ratio must be below the `MaxGearingRatio` threshold set to 0.5.

Only if all the thresholds are met or surpassed by the applicant, the app updates the loan request automatically with an _Accepted_ status. Otherwise, the application is updated with a _Rejected_ status. The app also caters for a scenario of programmatic errors that means a decision could not be made with a _UnderwritingFailure_ status.

### <input type="checkbox" unchecked/> Access additional resources

🗝️ You may want to enhance this simple working guide with some UI elements - why not use [Embedded Link](https://docs.codat.io/auth-flow/authorize-embedded-link) to seamlessly include our authorization journey into your app?

> 📊 If you are interested in underwriting models used by lenders in the industry, you can explore some recommended articles such as [Bigfoot Capital's](https://www.bigfootcap.com/revenue-based-financing/) blog on revenue-based finance or [Workweeks](https://workweek.com/2023/03/02/unlocking-lending-innovation) article on unlocking underwriting innovation.

Reviewer: ```suggestion
📊 If you are interested in underwriting models used by lenders in the industry, you can read through [Bigfoot Capital's](https://www.bigfootcap.com/revenue-based-financing/) blog on revenue-based finance or [Workweeks](https://workweek.com/2023/03/02/unlocking-lending-innovation) article on unlocking underwriting innovation.
```

💸 Lenders also use Assess to understand a business' liquidity via the [enhanced cash flow report](/assess/reports/enhanced-cash-flow-report/overview), or whether a business' accounts are accurate using both [data integrity](/assess/data-integrity) and the [audit report](/assess/reports/audit-report).

> 💸 Assess' additional features, like the , can help you expand the data points you use for loan assessments and automatic decisioning. 

Reviewer: ```suggestion
```
